ctfcli/utils/challenge.py

```diff
@@ -25,11 +25,15 @@ def sync_challenge(challenge):
         "category": challenge["category"],
         "description": challenge["description"],
         "type": challenge.get("type", "standard"),
-        "value": int(challenge["value"]),
+        "initial": int(challenge["value"]),
```

Comment:
Wouldn't this break other challenges that don't use an initial field? Initial should be specified in the spec file (perhaps a separate spec file) but not in the actual ctfcli code.

Reply:
Right, so if the challenge type is `dynamic`, then we'll send `initial` instead of `value`. That works?